its/src/test/java/com/sonar/it/csharp/DoNotAnalyzeTestFilesTest.java

```diff
@@ -54,7 +54,7 @@ public void with_csharp_only_test_should_not_populate_metrics() throws Exception
     assertThat(getMeasureAsInt(CSHARP_ONLY_TEST_PROJECT, "ncloc")).isNull();
 
     assertThat(buildResult.getLogsLines(l -> l.contains("WARN")))
-      .containsExactly("WARN: This C# sensor will be skipped, because the current solution contains only TEST files and no MAIN files. " +
+      .contains("WARN: This C# sensor will be skipped, because the current solution contains only TEST files and no MAIN files. " +
```

Comment:
Additional warnings, outside of our control, can be logged by the scanner. 

e.g. if `CodeCoverage.exe` is missing from the build machine (we don't have visual studio enterprise installed on the self hosted build machines).